src/kubernetes.cc

```diff
@@ -322,7 +316,7 @@ MetadataUpdater::ResourceMetadata KubernetesReader::GetContainerMetadata(
     k8s_container_name,
   };
 
-  if (container_status) {
+  if (container_status && container_status->Has("containerID")) {
```

Comment:
Why was this added? Do we directly depend on this property anywhere?

Reply:
Yes, right below. The deletion events apparently had a status without a container id.